HIPs/hlf-operator.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Hyperledger Fabric Operator Proposal
+parent: Hyperledger Improvement Proposals
+---
+
+# HIP Identifier
+[HIPs/hlf-operator.md](./hlf-operator.md)
+
+# Sponsor(s)
+
+- David Viejo [dviejo@kfs.es](mailto:dviejo@kfs.es)
+- Baohua Yang [baohua.yang@oracle.com](mailto:baohua.yang@oracle.com)
+- Aditya Joshi [adityaprakashjoshi1@gmail.com](mailto:adityaprakashjoshi1@gmail.com)
+
+# Abstract
+
+HLF Operator is a Kubernetes operator that helps simplify the deployments of Hyperledger Fabric in Kubernetes. It is based on the Hyperledger Fabric Helm charts and provides a Kubernetes native way to deploy and manage Hyperledger Fabric networks.
+
+# Context
+
+There are many ways to deploy Hyperledger Fabric, but most of the alternatives focus on Docker and manually performed deployments, not to mention there's no support to perform operations like:
+
+- Renewal of the certificates
+- Upgrades
+- Channel management
+
+HLF-operator solves this by introducing the concept of the Kubernetes operator, which is a program that runs within Kubernetes and extends it with functionality specific to HLF that makes managing networks way easier than other alternatives such as Docker-based.
+
+# Dependent Projects
+
+Hyperledger Fabric Operator only requires **Hyperledger Fabric** to run.
+
+# Motivation
+
+When building Hyperledger Fabric networks, there are several barriers to starting using and deploying Hyperledger Fabric:
+
+- How do I deploy a network in Kubernetes?
+- How do I manage the certificate authorities/peers/orderers in a production environment?
+- How do I renew the certificates?
+
+Several projects try to achieve the same goal, such as:
+
+- https://github.com/KompiTech/hyperledger-fabric-operator
+- https://github.com/hyperledger-labs/fabric-operator
+- https://github.com/raftAtGit/hl-fabric-operator
+
+# Status
+
+Proposal
+
+# Solution
```

Comment:
Please provide more information for the solution, since the project already exists and has been in labs for a while now. Check how the Bevel team proposed their solution https://github.com/hyperledger/hyperledger-hip/blob/gh-pages/HIPs/bevel.md#solution.

Reply:
What topics are usually explained there? Technical? Achitectural?

Reply:
High level goal and vision of the project can be captured here. Architecture diagrams and explanations around it can be very handy to achieve that.